winter/pagination/page/page_processor_resolver.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any
+
+from .page import Page
+from .page_processor import PageProcessor
+from ...output_processor import IOutputProcessorResolver
+
+
+class PageOutputProcessorResolver(IOutputProcessorResolver):
+
+    def __init__(self):
+        self._page_processor = PageProcessor()
+
+    def is_supported(self, body: Any):
+        return isinstance(body, Page)
+
+    def get_processor(self, body: Any):
```

Comment:
is to specify narrow return type hint (PageProcessor here) - correct? how do u think?